File: Python Exams/Basic Exam Collection/Basics Online Exam - 9 and 10 March 2019/05. Tennis Ranklist/05. Tennis Ranklist.py
tournaments_number = int(input())
start_points = int(input())

points_dict = {'W' : 2000, 'F' : 1200, 'SF': 720}
final_points = start_points
win = 0

for _ in range(tournaments_number):
    current_tournament = input()
    final_points += points_dict[current_tournament]
    if current_tournament == 'W': win +=1


average_points = (final_points - start_points) // tournaments_number
win_percent = win / tournaments_number * 100

print(f'Final points: {final_points}\nAverage points: {average_points}\n{win_percent:.2f}%')



'''
5
1400
F
SF
W
W
SF
'''

File: JS Exams/01.  Basics Exam Collection/Basics Online Exam - 9 and 10 March 2019/05. Tennis Ranklist/05. Tennis Ranklist.js
function tennisRanklist(arg) {
    let tournamentsNumber = Number(arg.shift())
    let startPoints = Number(arg.shift())

    const pointsDict = {'W': 2000,'F': 1200,'SF': 720}
    let finalPoints = startPoints
    let win = 0

    for (let i = 0; i < tournamentsNumber; i++) {
        let currentTournament = arg.shift()
        finalPoints += pointsDict[currentTournament]
        currentTournament === 'W'? win++ : null
    }

    let  averagePoints = (finalPoints - startPoints) / tournamentsNumber
    let  winPercent = win / tournamentsNumber * 100
    
    return `Final points: ${finalPoints}\nAverage points: ${~~averagePoints}\n${winPercent.toFixed(2)}%`
}

//console.log(tennisRanklist([5, 1400, 'F', 'SF', 'W', 'W', 'SF']))